toolchain/sem_ir/file.h

```diff
@@ -325,6 +333,10 @@ class File : public Printable<File> {
   // Single-entry/single-exit regions that are referenced as units, e.g. because
   // they represent expressions.
   ValueStore<ExprRegionId> expr_regions_;
+
+  // List of non-owning function declarations that are expected to be found in
+  // other libraries. Default to 0 size.
```

Comment:
Can you detail why you explicitly set 0? To be sure, the comment sounds a little like it's saying it changes `size()`, but my understanding of SmallVector is this is setting the stack-allocatable space to 0 so any entries would always end up on the heap.